ptero_shell_command/implementation/celery_tasks/shell_command.py

```diff
@@ -51,16 +57,19 @@ def run(self, command_line, umask, user, working_directory,
             return exit_code == 0
 
         except PreExecFailed as e:
+            LOG.warning('pre-exec failed: %s' % e.message)
```

Comment:
I don't know what meanings we associate with the various logging levels.  One convention is described on the Python logging [HOWTO](https://docs.python.org/2/howto/logging.html).  According to that convention, an INFO level message is a "Confirmation that things are working as expected," and WARNING is "an indication that something unexpected happened."

What